host/examples/rx_ascii_art_dft.cpp

```diff
@@ -169,20 +170,115 @@ int UHD_SAFE_MAIN(int argc, char *argv[]){
             ascii_art_dft::log_pwr_dft(&buff.front(), num_rx_samps)
         );
         std::string frame = ascii_art_dft::dft_to_plot(
-            lpdft, COLS, LINES,
+            lpdft, COLS, LINES-5,
             usrp->get_rx_rate(),
             usrp->get_rx_freq(),
             dyn_rng, ref_lvl
         );
+        
+        std::string border = std::string((COLS), '-');
 
         //curses screen handling: clear and print frame
         clear();
+        printw("%s", border.c_str());
+        printw("[f-F]req: %4.3f MHz   |   [r-R]ate: %2.2f Msps   |   [b-B]w: %2.2f MHz   |   [g-G]ain: %2.0f dB\n\n", freq/1e6, rate/1e6, bw/1e6, gain);
```

Comment:
Line too long, please limit to 80 characters (ish).
```suggestion
        printw("[f-F]req: %4.3f MHz   |   [r-R]ate: %2.2f Msps   |   "
                   "[b-B]w: %2.2f MHz   |   [g-G]ain: %2.0f dB\n\n",
                   freq/1e6, rate/1e6, bw/1e6, gain);
```
(I don't want to fiddle with editing code in Github's browser, but these should be indented to the correct level)